src/encode.c

```diff
@@ -1182,11 +1182,12 @@ PyImaging_Jpeg2KEncoderNew(PyObject *self, PyObject *args) {
     OPJ_PROG_ORDER prog_order;
     char *cinema_mode = "no";
     OPJ_CINEMA_MODE cine_mode;
+    char mct;
```

Comment:
Optional parameters should have a default value.
```suggestion
    char mct = 0;
```

Reply:
Done